lib/nadia/api.ex

```diff
@@ -11,8 +11,8 @@ defmodule Nadia.API do
   defp token, do: Application.get_env(:nadia, :token)
   defp recv_timeout, do: Application.get_env(:nadia, :recv_timeout, @default_timeout)
 
-  defp build_url(method, options \\ []) do
-    @base_url <> (options[:token] || token) <> "/" <> method
+  defp build_url(method, token) do
+    @base_url <> (token || token()) <> "/" <> method
```

Comment:
fill free to change the existing function name(`token`) for readability.
I prefer to `token || default_token`.